docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -0,0 +1,198 @@
+
+This means that one instance will be actively serving traffic, while the other one is on standby.
+
+##### Operate
+
+In the event of a total active region loss, the following data will be lost:
+
+- Batch Operations <!-- Do we have a docs link for this? -->
+
+##### Tasklist
+
+In the event of a total active region loss, the following data will be lost:
+
+- Assignments of tasks
+
+## Requirements
+
+- Camunda 8
+  - Minimum [Helm chart version](https://github.com/camunda/camunda-platform-helm) **9.3+**
+  - Minimum component images
+    - Elasticsearch **8.9+**
+    - Operate **8.5+**
+    - Tasklist **8.5+**
+    - Zeebe **8.5+**
+    - Zeebe Gateway **8.5+**
+- Two Kubernetes clusters
+  - OpenShift is not supported
+  - those need to be able to contact each other via, for example, VPC peering or similar means
+  - an [example implementation](<!-- TODO: Link -->) of two VPC peered Kubernetes clusters based on AWS EKS
+- Elasticsearch usage
+  - OpenSearch (both managed and self-managed) is not supported
+- Maximum RTT (round trip time) of 100 ms between the two Kubernetes clusters
+- Open Ports between the two Kubernetes clusters
+  - **9200** for Elasticsearch since Zeebe has to push data cross-region
+  - **26500** for the Zeebe gateway from client/worker communication
+  - **26501** for the Zeebe brokers and Zeebe gateway communication
+  - **26502** for the Zeebe brokers and Zeebe gateway communication
+- Only specific combinations of Zeebe broker counts and replication factors are supported
+  - `clusterSize` must be a multiple of **two** and a minimum of **four** to evenly distribute the brokers across the two regions
+  - `replicationFactor` must be **four** to ensure that the partitions are evenly distributed across the two regions
+  - `partitionCount` is not restricted and depends on your workload requirements, consider having a look at [understanding sizing and scalability behavior](../../../components/best-practices/architecture/sizing-your-environment.md#understanding-sizing-and-scalability-behavior)
+  - for further information and visualization of the partition distribution consider conducting the documentation about [partitions](../../../components/zeebe/technical-concepts/partitions.md)
+
+## Limitations
+
+- Camunda 8 must be installed with the [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm)
+  - alternative installation methods (for example with docker-compose installation) are not supported
+- Looking at the whole Camunda Platform, it's **active-passive**, while some key components are active-active
+  - meaning there's always one active and one passive region for serving active user traffic
+  - serving traffic to both regions will result in a detachment of the WebApps and users potentially observing different data in Operate and Tasklist
+- The customers operating their Camunda 8 setup are responsible for detecting a regional failure and executing the [operational procedure](<-- TODO: link -->)
```

Comment:
I think it's rather for Requirements (and deserves an extra mention in the `Disclaimer` on the top imo)

Reply:
added a disclaimer and moved on top